### PR TITLE
2023-04-19 TAC meeting notes

### PR DESCRIPTION
**Pull request by jfpanisset**

### CODE REVIEW

File: meetings/2023-04-19.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+[Video Conference Link](https://zoom.us/j/97880950229?pwd=V3ZGM0RiK0dDSi9Ja0Z1ZitJZVB3dz09)
+
+## Voting member attendance
+
+### Premier member representatives
+
+* [ ] Bill Roberts - Adobe Inc.
+* [ ] Brian Cipriano - Google LLC / OpenCue
+* [x] Cory Omand - The Walt Disney Studios
+* [ ] Eric Enderton - NVIDIA / DPEL
```
Comment (lgritz) [inline meetings/2023-04-19.md:17]: pretty sure Eric was there

File: meetings/2023-04-19.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+* [x] Michael B. Johnson - Apple Inc
+* [ ] Roy C Anthony - DNEG
+* [x] Sean C McDuffee - Intel Corporation
+* [ ] Sean O'Connell - Advanced Micro Devices (AMD)
+
+### Project representatives
+
+* [x] Carol Payne - OpenColorIO
+* [x] Cary Phillips - OpenEXR
+* [ ] Chris Kulla - Open Shading Language
```
Comment (lgritz) [inline meetings/2023-04-19.md:36]: And I think I saw Chris on the call?

Comment (jfpanisset) [reply meetings/2023-04-19.md:36]: Thanks for those, I swear there was a way to see the list of attendees for past meetings in the LFX console, but can't seem to find it right now. I still rely on taking a screenshot of the list of attendees at some arbitrary point in the meeting to make sure I didn't miss anyone at the end...

File: meetings/2023-04-19.md (hunk 1 above)
Comment (jfpanisset) [reply meetings/2023-04-19.md:17]: Added

**Pull request merged**